k8s/commander.sh

```diff
@@ -17,8 +17,16 @@ function deploy {
     DEPLOY_SECRETS=${5:-NO}
     K8S_NAMESPACE="sumo-${REGION_ENV}"
 
-    export KUBECTL_BIN="./regions/${REGION}/kubectl"
-    export KUBECONFIG="./regions/${REGION}/kubeconfig"
+    if [ -f "./regions/${REGION}/kubectl" ] ; then
+        KUBECTL_BIN="./regions/${REGION}/kubectl"
+    else
+        KUBECTL_BIN=$(which kubectl)
+    fi
+    export KUBECTL_BIN
+
+    if [ -f "./regions/${REGION}/kubeconfig" ] ; then
+        export KUBECONFIG="./regions/${REGION}/kubeconfig"
```

Comment:
You could move this export in line 22. Ifl line 20 is true then no need for this check.

Reply:
Well, line 20 checks for kubectl, this checks for a configuration file. 
I think it's safer this way because we do an extra check. Also this doesn't add extra complexity or anything like that